Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -163,12 +163,6 @@ private void UpdateControllerData(InteractionSourceState interactionSourceState)
 
                 // Devices are considered tracked if we receive position OR rotation data from the sensors.
                 TrackingState = (IsPositionAvailable || IsRotationAvailable) ? TrackingState.Tracked : TrackingState.NotTracked;
-
-                if (CameraCache.Main.transform.parent != null)
```

Comment:
We definitely need this, otherwise the controller position/rotation won't be in world space. Not everything that needs this info is parented under the camera parent, so we should transform into world space.

Reply:
I'm not keen on transforming if over and over again.

Let's annotate the property letting people know that it's local position, and or add a local position property to access it.

Reply:
> Not everything that needs this info is parented under the camera parent

Can you give a specific example we'd be using in beta?

Reply:
There are/will be solvers that depend on the positions of sources. I guess we can address it then.

Reply:
I'm open to the idea of  adding a Position/Rotation property for world space and a LocalPosition/LocalRotation proptery for local space.